LabExT/Tests/View/ExperimentWizard_test.py

```diff
@@ -222,17 +223,17 @@ def test_experiment_wizard(self):
 
         # randomly set InsertionLossSweep properties
         random_ilm_props = {
-            "wavelength start": random.randint(1460, 1550),
-            "wavelength stop": random.randint(1550, 1640),
-            "wavelength step": random.choice([1.0, 2.0, 5.0, 10.0, 20.0, 25.0, 50.0]),
+            "wavelength start": 1540,
+            "wavelength stop": 1560,
+            "wavelength step": 20,
```

Comment:
haha, why did you take out the random values?

Reply:
I thought I only want deterministic behaviour in my test :) can also reverse it. Plus, I guess this is more about testing the measurement InsertionLossSweep itself which would be better done in a different test.

Reply:
I agree with Nino